### PR TITLE
LPS-203320 Sitemap integration tests

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPS-203320

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/site/site-test/src/testIntegration/java/com/liferay/site/manager/test/SitemapManagerTest.java [hunk 1]
```diff
@@ -133,6 +164,12 @@ private void _assertSitemap(String uuid, String... urls) throws Exception {
 		String xml = _sitemapManager.getSitemap(
 			uuid, _group.getGroupId(), false, _themeDisplay);
 
+		if (urls.length == 0) {
```
Comment (Mikellorza) [inline modules/apps/site/site-test/src/testIntegration/java/com/liferay/site/manager/test/SitemapManagerTest.java:423]: What do you think to move this logic to another method to clarify the code? Something like this:

private void _assertEmtpySitemap(String uuid){
     Assert.assertEquals(StringPool.BLANK, _sitemapManager.getSitemap(
			uuid, _group.getGroupId(), false, _themeDisplay));
}

**Review by Mikellorza — CHANGES_REQUESTED** [review]

**Comment by Mikellorza** [comment]
Hi @lfbesada,

I added some comments.

thanks for the PR :)

**Comment by lfbesada** [comment]
Hi @Mikellorza Thanks for your feedback :heart: I have added it, could you please take a look?
Thanks

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
LGTM!!

thanks @lfbesada

**Comment by lfbesada** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**